### PR TITLE
installation of python/pip/setuptools is now handled in bootstrap script

### PR DESCRIPTION
**Pull request by lsh-0**
removes a lot of cruft for an ancient 'fix'

(depends on https://github.com/elifesciences/builder/pull/365)

### CODE REVIEW

File: elife/python.sls [hunk 1]
```diff
@@ -66,14 +66,7 @@ global-python-requisites:
 
 # DEPRECATED. removed after switch to 16.04
```
Comment (giorgiosironi) [inline elife/python.sls]: does the 16.04 part need any change given the shift of the installation into builder?

Comment (lsh-0) [reply elife/python.sls]: probably, I didn't look at it.

Comment (giorgiosironi) [reply elife/python.sls]: The 16.04 all seems backward compatible, but cross-check please

**Comment by giorgiosironi** [comment]
Several states in elife-bot-formula are using `requires` on these:

```
require:
    elife-bot--vagrant:                                      pip: global-python-requisites
 require:
    elife-bot--vagrant:                                      pkg: python-pip
```

I'll add them as empty states for backward compatibility.

**Comment by giorgiosironi** [comment]
They are also too specific (`pip`, `pkg`) so I'll have to amend the formula anyway first.

**Comment by giorgiosironi** [comment]
Java installation is failing and wrecking apt-get. Usually when Oracle removes old archives this happens.

**Comment by giorgiosironi** [comment]
Ok, the Java installation is a recurring problem that happens every time Oracle removes the old versions of binaries. Here is some links to work around it @lsh-0 if you happen to be around, don't know your schedule.

The stuff to uncomment: https://github.com/elifesciences/builder-base-formula/blob/master/elife/java8.sls#L31-L37
What to put in for the numbers (untested): 
- https://twitter.com/behemphi/status/953372016677146630
- https://twitter.com/behemphi/status/953371839937630208


(and then we move to OpenJDK or containers down the road?)

**Comment by lsh-0** [comment]
hey @giorgiosironi , yes I'm about today but not tomorrow. I'll take a look at this java stuff.

I think elife-bot is the only one still using rmrf_enter as well, an opportunity there to remove some complexity. I didn't touch it this time around because `pip.installed` should be working regardless of what it's installing, but food for thought.

**Comment by lsh-0** [comment]
the java stuff appears to work fine on fresh instances (tested on vagrant) and subsequent highstates, but I had a problem with dpkg needing to be reconfigured on existing instances. I'll push my changes and see if we can replicate, but worst case we have to do some manual wrangling on java enabled instances

**Comment by lsh-0** [comment]
@giorgiosironi , I'm not going to push any changes up regarding the java stuff, it appears to be working now. I can build a fresh an instance with it as-is. The dpkg problems I was having before may have been due to uncommenting all that `sed` stuff.

**Comment by lsh-0** [comment]
four failed states, all unrelated to java. 

> I'll add them as empty states for backward compatibility.

these failed because these states weren't found. I'll add them in now.

**Comment by lsh-0** [comment]
looking good! merge if you're happy

**Pull request merged**